src/components/Header/Header.js

```diff
@@ -0,0 +1,29 @@
+import React,  {useState } from 'react';
+import { Link } from 'react-router-dom';
+import Hamburger from "hamburger-react";
+import './_Header.scss';
+
+const Header = () => {
+  const [isOpen, setOpen] = useState(false)
+  
+  return (
+    <header className='mobileHeader'>
+      <Hamburger toggled={isOpen} toggle={setOpen} />
```

Comment:
This looks great!

Reply:
Oh my gosh, this npm package is so worth it.